site/components/Navigation/index.tsx

```diff
@@ -58,47 +64,345 @@ export const Navigation: FC<Props> = ({
+          subMenu={[
+            <NavItemDesktop
+              name={t({
+                message: "BUY KLIMA",
+                id: "shared.buy",
+              })}
+              key="buy klima"
+              href={createLinkWithLocaleQuery(urls.buy, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "STAKE KLIMA",
+                id: "shared.stake",
+              })}
+              key="stake"
+              href={createLinkWithLocaleQuery(urls.stake, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "BOND KLIMA",
+                id: "shared.bond",
+              })}
+              key="bond"
+              href={createLinkWithLocaleQuery(urls.bonds, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "WRAP SKLIMA",
+                id: "shared.wrap",
+              })}
+              key="wrap"
+              href={createLinkWithLocaleQuery(urls.wrap, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "OFFSET",
+                id: "shared.offset",
+              })}
+              key="offset"
+              href={createLinkWithLocaleQuery(urls.offset, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "INFO",
+                id: "shared.info",
+              })}
+              key="info"
+              href={createLinkWithLocaleQuery(urls.info, locale)}
+            />,
+          ]}
         />
         <NavItemDesktop
-          url={"/infinity"}
           name={t({ message: "Infinity", id: "shared.infity" })}
           active={activePage === "Infinity"}
+          subMenu={[
+            <NavItemDesktop
+              name={t({
+                message: "INTRODUCTION",
+                id: "shared.intoduction",
+              })}
+              key="intro"
+              href="/infinity"
+            />,
+            // these can be uncommented with resources page reqork
+            // <NavItemDesktop
+            //   label={t({
+            //     message: "HOW TO PLEDGE",
+            //     id: "shared.how_to_pledge",
+            //   })}
+            //   key="how pledge"
+            //   href="/blog"
+            //   className={styles.navMain_DesktopMenuItem}
+            // />,
+            // <NavItemDesktop
+            // label={t({
+            //   message: "LEADER BOARD",
+            //   id: "shared.leader_board",
+            // })}
+            //   key="leader board"
+            //   href="/blog"
+            //   className={styles.navMain_DesktopMenuItem}
+            // />,
+          ]}
         />
         <NavItemDesktop
           url={urls.loveletter}
           name={t({ message: "Love Letters", id: "shared.loveletters" })}
+          href={createLinkWithLocaleQuery(urls.loveletter, locale)}
```

Comment:
`createLinkWithLocaleQuery` is only needed if you link to APP.
Loveletter is not an APP URL (does not start with `https://app.klimadao.finance/ .....`)
Same goes for the other links in this file.
Only use `createLinkWithLocaleQuery` if you link to APP.

`createLinkWithLocaleQuery` adds the current language to the URL when directing to the APP.
This ensures APP is rendered with the same language the user chose on SITE.

Reply:
Oops I did everything with a klimadao url. fixed.